tests/test_registrations/test_archiver.py

```diff
@@ -536,6 +536,13 @@ def test_archive_success_with_components(self):
                 )
                 patch.start()
                 patches.append(patch)
+                n_patch = mock.patch.object(
```

Comment:
For the tests that you had to modify their mocking with, do you have contentious tests that will test both the positive and negative conditions to show that it's not just covering up the failing behavior but is definitely working? Not that I'm suspicious of Mock or anything.